python/lsst/ts/scheduler/coldstart/database.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
I'm not sure why you created a class to handle this kind of things. I would recommend using [pandas](https://pandas.pydata.org) to connect and read the database instead.